app/views/pregnancies/_call_log.html.erb

```diff
@@ -3,14 +3,29 @@
 		<h1 class="border-bottom title"> Call Log</h1>
 		<table class="table border-side">
 		<tr>
-			<th>Time Stamp</th>
+			<th>Date</th>
+			<th>Time</th>
 			<th>Result</th>
 			<th>user</th>
 		</tr>
+		<div class="btn btn-primary" id="toggle-call-log">View all calls</div>
 		<tbody>
-			<td><%= pregnancy.initial_call_date %></td>
-			<td><%= pregnancy.status %></td>
-			<td><%= pregnancy.user %></td>
+			<% @pregnancy.recent_calls.each do |call| %>
+				<tr>
+					<td><%= call.created_at.getlocal.strftime("%-m/%-d") %></td>
+					<td><%= call.created_at.getlocal.strftime("%l:%M %P") %></td>
+					<td><%= call.status %></td>
+					<td><%= "user" %></td>
+				</tr>
+			<% end %>
+			<% @pregnancy.old_calls.each do |call| %>
+				<tr class="old-calls hidden">
+					<td><%= call.created_at.getlocal.strftime("%-m/%-d") %></td>
```

Comment:
this reminds me, we'll want to figure out a unified way to display datestamps and timestamps. Going to create an issue for this and probably develop a convenience method for it.